pkg/controller/worker/machines.go

```diff
@@ -44,6 +46,8 @@ const (
 	maxGcpLabelCharactersSize = 63
 	// ResourceGPU is the GPU resource . It should be a non-negative integer.
 	ResourceGPU v1.ResourceName = "gpu"
+	// CSIDiskDriverTopologyKey is the label on persistent volumes that represents availability by zone.
+	CSIDiskDriverTopologyKey = "topology.gke.io/zone"
```

Comment:
Out of convention, I would probably move this key to https://github.com/gardener/gardener-extension-provider-gcp/blob/master/pkg/gcp/types.go